cgi-bin/network.pl

```diff
@@ -3713,6 +3730,12 @@ sub viewStorage
 		my $used  = $D->{hrStorageUnits} * $D->{hrStorageUsed};
 		my $free  = $total - $used;
 
+		# check if available units present in inventory else calculate it.
+			if (defined $D->{hrStorageAvailUnits}){
+				$available_data = $D->{hrStorageUnits} * $D->{hrStorageAvailUnits};
+			}else{
+				$available_data = $free + $cached_data + $buffers_data;
```

Comment:
we do not want this calculation in 3 places, if the value is not present just don't show it